frontend/e2e-tests/page-objects/input/SaveFormPgObj.ts

```diff
@@ -4,13 +4,21 @@ import { InputBasePage } from "./InputBasePgObj";
 
 export class SaveFormPage extends InputBasePage {
   readonly heading: Locator;
+  readonly summaryText: Locator;
+  readonly summaryList: Locator;
   readonly save: Locator;
 
   constructor(page: Page) {
     super(page);
 
     this.heading = page.getByRole("heading", { level: 2, name: "Controleren en opslaan" });
-
+    this.summaryText = page.getByTestId("save-form-summary-text");
+    this.summaryList = page.getByTestId("save-form-summary-list");
     this.save = page.getByRole("button", { name: "Opslaan" });
   }
+
+  summaryListItem(text: string) {
+    const li = this.summaryList.locator("li", { hasText: text });
```

Comment:
Better to use `getByRole()`: `this.summaryList.getByRole("listitem", { "name": hasText})`

Reply:
I couldn't get this to work, maybe because the text is inside an <article> tag?

Reply:
The `<article>` is not the problem, but the design of listitem is, see: https://github.com/testing-library/dom-testing-library/issues/1006